samples/Dantooine/Dantooine.Client/Client/Shared/NavMenu.razor

```diff
@@ -0,0 +1,49 @@
+﻿<div class="top-row pl-4 navbar navbar-dark">
+    <a class="navbar-brand" href="">Blazor OpenIddict BFF Cookies</a>
+    <button class="navbar-toggler" @onclick="ToggleNavMenu">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+</div>
+
+<div class="@NavMenuCssClass" @onclick="ToggleNavMenu">
+    <ul class="nav flex-column">
+        <AuthorizeView>
+            <Authorized>
+
+                <li class="nav-item px-3">
+                    <NavLink class="nav-link" href="" Match="NavLinkMatch.All">
+                        <span class="oi oi-home" aria-hidden="true"></span> Home
+                    </NavLink>
+                </li>
+                <li class="nav-item px-3">
+                    <NavLink class="nav-link" href="directapi">
+                        <span class="oi oi-list-rich" aria-hidden="true"></span> Direct API
+                    </NavLink>
+                </li>
+                <li class="nav-item px-3">
+                    <NavLink class="nav-link" href="dantooineapi1">
+                        <span class="oi oi-list-rich" aria-hidden="true"></span> Dantooine Api1
+                    </NavLink>
+                </li>
+  
+            </Authorized>
+            <NotAuthorized>
+                <li class="nav-item px-3">
+                    <p class="whiteColor">no access</p>
```

Comment:
Should it say something like `Log in to view the menu` so it's clearer? It confused me a bit when running the app.

Reply:
updated